sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -270,6 +270,7 @@ public ContactDirectorySteps(
         "I open the last created contact",
         () -> {
           searchAfterContactByMultipleOptions(apiState.getCreatedContact().getUuid());
+          TimeUnit.SECONDS.sleep(3); // waiting for page loaded
```

Comment:
waitUntilAListOfWebElementsAreNotEmpty maybe this one can be a fix, to wait until grid will contain results, so if takes more than 3 seconds, we can wait 20 with default method